src/error-display/error-display-data.tsx

```diff
@@ -286,16 +286,18 @@ export const ERROR_DISPLAY_DATA = new Map<
         "unsupported-browser",
         {
             img: imgAttributeHelper(ImgPaths["unsupported-browser"]),
-            title: "Error state",
-            description: "A generic error state illustration.",
+            title: "Browser not supported",
+            description:
+                "LifeSG works best with the latest version of Chrome, Edge, Firefox or Safari. If you continue with your browser, you may run into problems.",
```

Comment:
the old copy for unsupported browser is
> To use LifeSG, download the latest version of Chrome, Edge, Firefox or Safari.

designers wanted to align the error messages, the latest default copy is [here](https://www.figma.com/file/us90jOpWBahsqK2VAluPvD/Flagship-Design-System?type=design&node-id=18639%3A33558&mode=design&t=55abRmuH8isXsVsH-1)

could you override the copy on the web repo itself?